### ISSUE TITLE
Remote Desktop with Shadow mode

### ISSUE THREAD

**Issue by fanynek86** [Ylianst/MeshCentral#6552]
Hello gentlemen!

I would like to ask you...
Wouldn't it be possible to add the "Shadow mode" rdp startup parameter to MeshCentral? To use RDP like this:

https://www.informaticar.net/remote-desktop-with-shadow-mode-in-windows/

**Comment by si458** [comment]
This is already sort of possible.
Go to ur device then desktop,
Untick the Input checkbox at the bottom,
Then click connect and it should show a drop down of rdp sessions, then u click which one to connect too
Once connected if u need to control there machine, click the input checkbox again :)

**Comment by fanynek86** [comment]
I'm sorry, I've clicked RDP almost everywhere, but I don't see this option anywhere. As for Web-RDP, even when "input" is unchecked, it wants to switch the current user.

**Comment by si458** [comment]
@fanynek86 you click the normal `connect` button as show in my example video
just remember as i said to untick the `input` box at the bottom first, otherwise you will control there session

https://github.com/user-attachments/assets/a1063283-59f2-4643-963e-29f8bb08a93c

**Comment by fanynek86** [comment]
We don't seem to understand each other. I would like to use Shadow mod, to connect via "Mstsc.exe /shadow:<Session ID> /v:<Computer name or IP address>". So that the user does not switch. As it is in this link: 
https://www.informaticar.net/remote-desktop-with-shadow-mode-in-windows/

The user and remote support can then control the PC simultaneously.

**Comment by si458** [comment]
@fanynek86 yes above in the video is how you do it, but we dont call it 'Shadow Mode',
when a user connects by RDP to a machine, the staff can then load up that machine in meshcentral, click the connect button, and then click which user they wanted to control/shadow!
but this only works IF you have the meshagent installed on the remote device
it wont work if you use local groups or relay groups

**Comment by fanynek86** [comment]
Sorry for not commenting for so long. I am talking about classic RDP, which starts the MeshCentral Router and then we can use it to connect to the target station. The mentioned "Shadow mode" connects to the current logged-in user and works like VNC, for example. Here's a preview, for your understanding:

https://www.youtube.com/watch?v=d3Pa7adKfQ4

Here are more detailed instructions:

https://blog.bitsadmin.com/spying-on-users-using-rdp-shadowing

**Comment by si458** [comment]
oh right! well in that case this issue is more for the MeshCentralRouter repo! so ill move the issue there
as the RDP link just opens up the meshCentralRouter which then inturn opens up the RDP app on your windows device
so the code would need changing in there,
also remember when you SHADOW a session, you need to know what session before hand to shadow as you could have multiple sessions
this is where the meshcentral web ui has the advantage as it already knows the sessionids
